invenio_github/config.py

```diff
@@ -62,39 +62,32 @@
 GITHUB_REFRESH_TIMEDELTA = timedelta(days=1)
 """Time period after which a GitHub account sync should be initiated."""
 
-GITHUB_RELEASE_CLASS = 'invenio_github.api:GitHubRelease'
+GITHUB_RELEASE_CLASS = "invenio_github.api:GitHubRelease"
 """GitHubRelease class to be used for release handling."""
 
-GITHUB_METADATA_FILE = '.invenio.json'
+GITHUB_METADATA_FILE = ".invenio.json"
 """File with extra metadata stored in GitHub repository."""
 
-GITHUB_CITATION_FILE = 'CITATION.cff'
+GITHUB_CITATION_FILE = "CITATION.cff"
 """File with citation metadata stored in GitHub repository."""
 
-GITHUB_DEPOSIT_CLASS = 'invenio_deposit.api:Deposit'
-"""Deposit class that implements a `publish` method."""
-
-GITHUB_RECORD_SERIALIZER = None
-"""Serializer to use to display record metadata."""
-
-GITHUB_PID_FETCHER = 'recid'
-"""PID Fetcher for Release records."""
-
-GITHUB_TEMPLATE_INDEX = 'invenio_github/settings/index.html'
+GITHUB_TEMPLATE_INDEX = "invenio_github/settings/index.html"
```

Comment:
So I left them - even if they are failing to load - because we're working on the UI on top of this PR. So I made sure that the building worked (e.g. added webpack configs) and tested the UI endpoints (e.g. they return the data needed for the templates to work).